### ISSUE TITLE
[ActionButton] Label styles

### ISSUE THREAD

**Issue by NewOldMax** [xotahal/react-native-material-ui#75]
As I see, currently is not possible to set styles for label's text
````
    renderLabelAction = (styles, icon, label) => (
        <View key={icon} style={styles.speedDialActionContainer}>
            <View style={styles.speedDialActionLabelContainer}>
                <Text>{label}</Text>
            </View>
            {this.renderAction(styles, icon)}
        </View>
    )
````
For example, I want to change label color to white, but currently I only can change label container styles.
![image](https://cloud.githubusercontent.com/assets/1328473/21849191/90032026-d815-11e6-9643-9be30f6dbee2.png)
Should I create PR for this?

**Comment by xotahal** [comment]
It would be great ;) There also should be somthing like `speedDialActionLabel` style object in `getTheme` to default setting of that style. What do you think?

**Comment by NewOldMax** [comment]
Sure, I was planning to do so.
Can you look also at #66 ?

**Comment by xotahal** [comment]
Yeah, I need to do that :/ Sorry .. need some collaborators ..

**Comment by NewOldMax** [comment]
closed by #76
